conda-recipe/meta.yaml

```diff
@@ -47,6 +47,7 @@ outputs:
         # need to update our packages to be compatible with pandas 2 API
         - pandas
         - psutil
+        - pydantic 2.*
```

Comment:
I'd consider a pin to the minor version. pydantic is still noticibly changing

Reply:
after discussion with @FynnBe, we'll keep the pin as is. The `pydantic` features used so far in ilastik are _very_ basic. Also pinning it further would require some coordination with `bioimageio` dependencies.